Workbooks/WorkbooksMetadata.json

```diff
@@ -1442,5 +1442,18 @@
       "templateRelativePath": "ADXvsLA.json", 
       "subtitle": "",
       "provider": "Azure Sentinel Community"
-    }     
+    },
+    {
+      "workbookKey": "ProofPointThreatDashboard",
+      "logoFileName": "",
+      "description": "Provides an overview of email threat activity based on log data provided by ProofPoint",
+      "dataTypesDependencies": [ "ProofpointPOD_message_CL", "ProofpointPOD_maillog_CL", "ProofPointTAPClicksBlocked_CL", "ProofPointTAPClicksPermitted_CL", "ProofPointTAPMessagesBlocked_CL", "ProofPointTAPMessagesDelivered_CL" ],
+      "dataConnectorsDependencies": ["Proofpoint TAP", "ProofpointPOD"],
```

Comment:
@reprise99 : Connector id for Proofpoint TAP is incorrect, Please update it to **"ProofpointTAP"**

Reply:
Apologies, completed